src/main/java/frc/robot/subsystems/LightsSubsystem.java

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Relay.Value;
+import edu.wpi.first.wpilibj.Relay;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class LightsSubsystem extends SubsystemBase {
+    int teamColor; // 1 is blue, 2 is red
+    boolean amplify = false;
+    boolean coopertition = false;
+    Timer timer = new Timer();
+
+    /** Creates a new LightsSubsystem. */
+    public LightsSubsystem() {
+        timer.reset();
+    }
+
+    private static Relay green = new Relay(Constants.ElectronicsPorts.green);
+    private static Relay red = new Relay(Constants.ElectronicsPorts.red);
+    private static Relay blue = new Relay(Constants.ElectronicsPorts.blue);
+
+    
+    @Override
+    public void periodic(){
+        if (DriverStation.getAlliance().get().toString() == "Blue") {
+            teamColor = 1;
+        } else if (DriverStation.getAlliance().get().toString() == "Red") {
+            teamColor = 2;
+        }
```

Comment:
ideally just compare DriverStation.getAlliance() == Alliance.Blue because enum Values might Change and thus the strings will change. Some compile environments will also break this